cups/ppd-cache.c

```diff
@@ -1427,6 +1452,10 @@ _ppdCacheCreateWithPPD(ppd_file_t *ppd)	/* I - PPD file */
 
   if ((ppd_attr = ppdFindAttr(ppd, "APPrinterPreset", NULL)) != NULL)
   {
+   /*
+    * "Classic" Mac OS approach
```

Comment:
Use the right name ("macOS") if you are going to comment on this. But keep in mind that we support APPrinterPreset on Linux/BSD/etc. as well (exposed as job-presets-supported for IPP Everywhere).

Reply:
Sorry, I could reword it into simply `"Classic" approach`.